android/java/org/chromium/chrome/browser/crypto_wallet/fragments/SwapBottomSheetDialogFragment.java

```diff
@@ -100,7 +100,8 @@ public void onClick(View view) {
 
         if (!JavaUtils.anyNull(mWalletModel, mNetworkInfo)) {
             BuySendSwapActivity.ActivityType finalActivityType = activityType;
-            mWalletModel.getNetworkModel().setNetwork(
+            // TODO(apaymyshev): buy/send/swap should be decoupled from panel selected network.
+            mWalletModel.getNetworkModel().setDefaultNetwork(
```

Comment:
It should be with set or get network with `mOrigin`, setNetwork in networkmodel internally use `mOrigin` to set network per origin.

Reply:
This is wallet ui, so should set default network